### ISSUE TITLE
Issue - Cannot find module 'keyvajs' or its corresponding type declarations. ts

### ISSUE THREAD

**Issue by bonnerwork** [paul-go/Keyva#1]
I got this issue while try to use this package with my next.js project.

**Comment by paul-go** [comment]
Yes, this has been reported before. Keyva doesn't work with ES modules just yet (I don't use them). I'll add that at some point. Thanks for reporting the issue, I'll keep this open until I get it fixed.
